tatanka/tatanka.go

```diff
@@ -122,6 +126,13 @@ type Config struct {
 
 	// TODO: Change to whitelist
 	WhiteList []BootNode
+
+	FiatOracleConfig
+}
+
+type FiatOracleConfig struct {
+	fiatrates.Config
+	FiatRateBroadcastInterval int64 `long:"fiatratebroadcastinterval" description:"Specify the minutes it'll take before the server notifies all subscribed clients with fiat rates (max allowed = 10). Set to zero to disable the fiat rate oracle."`
```

Comment:
Could also specify the default.

Reply:
Will change disable to -1 to allow us to set a default if a zero value is provided for this field.